docs/references/http-websocket-apis/public-api-methods/subscription-methods/subscribe.md

```diff
@@ -180,6 +181,7 @@ The fields from a ledger stream message are as follows:
 | `ledger_hash`       | String - [Hash][]         | The identifying hash of the ledger version that was closed. |
 | `ledger_index`      | Number - [Ledger Index][] | The ledger index of the ledger that was closed. |
 | `ledger_time`       | Number                    | The time this ledger was closed, in [seconds since the Ripple Epoch][] |
+| `network_id`        | Number                    | The XRPL network of this stream. |
```

Comment:
Add a "New in: rippled 2.6.0" badge. Also, it would be good to link a reference of which network IDs correspond to which networks, if we had one. (The "Parallel Networks" page seems like a good candidate, but doesn't have that information; the "Connect Your rippled to a Parallel Network" guide sort of has them, though it uses the shortcut names rather than the actual numeric values in the config examples.)

Reply:
I found a good link that lists out the network and networkID in the transaction common fields page.